src/app/charity/charity.component.spec.ts

```diff
@@ -9,7 +9,7 @@ describe('CharityComponent', () => {
   let component: CharityComponent;
   let fixture: ComponentFixture<CharityComponent>;
 
-  beforeEach(waitForAsync(() => {
+  beforeEach(() => {
```

Comment:
see https://dev.to/this-is-angular/simplifying-angular-testbed-setup-by-converting-to-synchronous-test-hooks-19cl

Reply:
Ah, nice!